src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int SECOND_ELEMENT = 1;
+    private static final String TXT_SUPPLY = "supply";
+    private static final String TXT_BUY = "buy";
+    private static final String TXT_RESULT = "result";
+    private static final String TXT_SEPARATOR = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        try {
+            File firstFile = new File(fromFileName);
+            File secondFile = new File(toFileName);
+
+            int buy = 0;
+            int supply = 0;
+
+            List<String> stringList = Files.readAllLines(firstFile.toPath());
```

Comment:
The `readAllLines` method can lead to OutOfMemoryError if the file is too large. Consider using `Files.lines` with a try-with-resources statement to read and process the file line by line.